vulkan/vulkan_utilities.cc

```diff
@@ -12,6 +12,8 @@ namespace vulkan {
 bool IsDebuggingEnabled() {
 #ifndef NDEBUG
   return true;
+#elif OS_FUCHSIA && (FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG)
```

Comment:
This will fail if this file or one of its includes does not include "flutter/fml/build_config.h". Please include it explicitly in this TU.

Reply:
Done